.docker/devdb.sh

```diff
@@ -1,13 +1,20 @@
 #!/bin/bash
 set -e
 
-psql "postgresql://$POSTGRES_USER@:5432/$POSTGRES_DB" -v ON_ERROR_STOP=1 <<-EOSQL
+until psql -U $POSTGRES_USER -c '\q'; do
+  >&2 echo "Postgres is unavailable - waiting..."
+  sleep 1
+done
+
+psql -U $POSTGRES_USER -v ON_ERROR_STOP=1 <<-EOSQL
+  DROP DATABASE IF EXISTS conduit_dev;
+  DROP USER IF EXISTS conduit_dev;
   CREATE USER conduit_dev WITH PASSWORD '';
   CREATE DATABASE conduit_dev;
   GRANT ALL PRIVILEGES ON DATABASE conduit_dev TO conduit_dev;
 EOSQL
 
-psql -v ON_ERROR_STOP=1 "postgresql://$POSTGRES_USER@:5432/conduit_dev" <<-EOSQL
```

Comment:
I prefer the full URL for two reasons:

1. Explicit is better than implicit.
2. If I need to inspect the database with pgqwb/psql/whatever, I open up `Makefile`, copy the URL and paste it into console. Done. After this change, I would have to manually construct the URL.

Reply:
I remember I read this argument once before. Probably in this app... 😄